modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/jaxrs/jaxb/JaxbProviderDeserializationSecurityCheckTestCase.java

```diff
@@ -56,15 +64,18 @@
  */
 @RunWith(Arquillian.class)
 @RunAsClient
-//@apAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/jaxrs/src/main/java","modules/testcases/jdkAll/Eap7/jaxrs/src/main/java","modules/testcases/jdkAll/Eap71x/jaxrs/src/main/java","modules/testcases/jdkAll/Eap70x/jaxrs/src/main/java"})
+@EapAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/jaxrs/src/main/java","modules/testcases/jdkAll/Eap7/jaxrs/src/main/java","modules/testcases/jdkAll/Eap71x/jaxrs/src/main/java","modules/testcases/jdkAll/Eap70x/jaxrs/src/main/java"})
```

Comment:
I think that "modules/testcases/jdkAll/Eap7/jaxrs/src/main/java" and "modules/testcases/jdkAll/Eap70x/jaxrs/src/main/java" should be removed as fasterxml version is not updated for these branches.

Reply:
You're right, thanks. Done.